db/model/aspect.js

```diff
@@ -110,6 +110,11 @@ module.exports = function aspect(seq, dataTypes) {
         },
       },
     },
+    tags: {
```

Comment:
All the validations are in the API layer. Doing it in the db layer will be redundant, I guess.